website/datadog.erb

```diff
@@ -34,15 +37,9 @@
             <li<%= sidebar_current("docs-datadog-resource-monitor") %>>
               <a href="/docs/providers/datadog/r/monitor.html">datadog_monitor</a>
             </li>
-            <li<%= sidebar_current("docs-datadog-resource-screenboard") %>>
```

Comment:
We wanted to follow the same approach we did for https://docs.datadoghq.com/api/#dashboards when we rolled out the new API to achieve two goals:
- Push users to use the new resource while still having access to the old resources
- Minimize confusion that might be caused by having three related resources accessible from the main page

That said, I don't have a strong opinion about that so if you still think we should keep the links I am happy add them back. let me know.

Reply:
I think it would be really confusing if the documentation just disappeared one day without the users understanding why. There's no good way in TF provider docs to go back to docs of older versions, so keeping the links there and putting the disclaimers there for screenboard and timeboard resources seems like the best solution to me.

Reply:
Since these won't be `deprecated` in the next immediate provider version, I also think we should keep the links. Once we start deprecation on the resources we should remove the links and log a notice. For now I think we should just leave a note at the top of each screenboard/timeboard page.

You can even use this to make it a big note `!>` at the top of the timeboard/screenboard pages. https://github.com/hashicorp/terraform-website#callouts (Just tested locally and confirmed it works. There is a `make website` command that will spin up a local copy of the site in docker)

Reply:
Sure. I added the links back. Thank you both.